src/go/languageservice/workspace.go

```diff
@@ -30,334 +30,288 @@ type WorkspaceID int
+    defer w.l.Unlock()
+    delete(w.Workspaces, key)
 }
 
 type Workspace struct {
-	Callbacks []WorkspaceUpdateCallback
-
-	// TODO: to enable incremental-ish behavior, I have separate file sets per file.
-	// The idea is that we can create a combined file set at a point in time for type
-	// checking if needed. Not sure how idomatic this is...
-	Files  map[string]*workspaceDocument
-	l      *sync.RWMutex
-	Errors []error
+    Callbacks []WorkspaceUpdateCallback
+
+    // TODO: to enable incremental-ish behavior, I have separate file sets per file.
+    // The idea is that we can create a combined file set at a point in time for type
+    // checking if needed. Not sure how idomatic this is...
+    Files  map[string]*workspaceDocument
+    l      *sync.RWMutex
+    Errors []error
 }
 
 func (ws *Workspace) SetWorkspaceDocument(key string, value *workspaceDocument) {
-	ws.l.Lock()
-	defer ws.l.Unlock()
-	ws.Files[key] = value
+    ws.l.Lock()
+    defer ws.l.Unlock()
+    ws.Files[key] = value
 }
 
 func (ws *Workspace) GetWorkspaceDocument(key string) (*workspaceDocument, error) {
-	ws.l.RLock()
-	defer ws.l.RUnlock()
-	document, ok := ws.Files[key]
-	if !ok {
-		return nil, errors.New(" workspaceDocument not found")
-	}
-	return document, nil
+    ws.l.RLock()
+    defer ws.l.RUnlock()
+    document, ok := ws.Files[key]
+    if !ok {
+        return nil, errors.New(" workspaceDocument not found")
+    }
+    return document, nil
 }
 func (ws *Workspace) DeleteWorkspaceDocument(key string) {
-	ws.l.Lock()
-	defer ws.l.Unlock()
-	delete(ws.Files, key)
+    ws.l.Lock()
+    defer ws.l.Unlock()
+    delete(ws.Files, key)
 }
 
 // CreateNewWorkspace creates a Workspace and returns a unique Id
 // for it.
 func CreateNewWorkspace() WorkspaceID {
-	workspace := Workspace{
-		Callbacks: nil,
-		Files:     make(map[string]*workspaceDocument, 0),
-		l:         new(sync.RWMutex),
-		Errors:    nil,
-	}
+    workspace := Workspace{
+        Callbacks: nil,
+        Files:     make(map[string]*workspaceDocument, 0),
+        l:         new(sync.RWMutex),
+        Errors:    nil,
+    }
 
-	// TODO: there's a hypothetical concern with int wrap around as we open + close workspaces.
-	thisManagerID := WorkspaceID(nextManagerID)
-	manager.SetManager(thisManagerID, &workspace)
+    // TODO: there's a hypothetical concern with int wrap around as we open + close workspaces.
+    thisManagerID := WorkspaceID(nextManagerID)
+    manager.SetManager(thisManagerID, &workspace)
 
-	nextManagerID++
+    nextManagerID++
 
-	return thisManagerID
+    return thisManagerID
 }
 
 // CloseWorkspace frees an open workspace.
 func (id WorkspaceID) CloseWorkspace() {
-	delete(manager.Workspaces, id)
+    delete(manager.Workspaces, id)
 }
 
 // QueueFileParse queues the reparse of a file. Reparse is signaled to the caller
 // via a WorkspaceUpdateCallback.
 func (id WorkspaceID) QueueFileParse(fileName string, reader io.Reader, versionId uintptr) *error {
-	if reader == nil {
```

Comment:
Doesn't the Go linter prefer tabs? I've been of the opinion _when in Rome..._

Reply:
I'm ok with switching to spaces so long as we can put the right _.editorconfig_ or whatever is needed to keep the files consistent in the project.

Reply:
idk actually. I never changed the default settings of the builtin to Goland.

Reply:
In Visual Studio + VS Code we can configure these settings usually with .EditorConfig files. We can probably for Golang too.

Reply:
Yea for sure. Actually, I looked a bit closer and Goland is pulling from my IDE settings repository that's configured for other languages and I think it might have gotten messed up. I'll configure it before the next commit.

Reply:
No worries, this is a low-key side project. There are no standards :)